app/elm/View.elm

```diff
@@ -242,16 +242,11 @@ printComment comment now model =
         ]
 
 
-printResponses : Maybe Responses -> Maybe Date.Date -> Model -> Html Msg
+printResponses : Responses -> Maybe Date.Date -> Model -> Html Msg
 printResponses responses now model =
-    case responses of
-        Just responseList ->
-            ul [] <|
-                List.map (\c -> printComment c now model) <|
-                    unwrapResponses responseList
-
-        Nothing ->
-            nothing
+    ul [] <|
+        List.map (\c -> printComment c now model) <|
+            unwrapResponses responses
```

Comment:
Thanks again for taking a look at this. I tried that before, but don't understand why it would throw a `Cannot find pattern 'Responses'` naming error. Responses is exposed on line 4, so I basically gave up and unwrapped it.

Reply:
The `cannot find pattern` error comes from the fact that you are importing the _type_ `Responses`, but none of its constructors. This can be fixed by changing the import line to one of these:

```elm
-- this version exposes the `Responses` type _and_ the constructor `Responses`. This
-- import style is useful when you want to limit which constructors are pulled in
import Data.Comment exposing (Comment, Responses(Responses))

-- this version exposes all constructors of the `Responses` type
import Data.Comment exposing (Comment, Responses(..))
```

Either way is identical in your scenario since you have a single constructor.

Reply:
Oh yes! That makes sense. Working and fixed now. Thanks.